src/Converter/convertToTile.js

```diff
@@ -45,12 +72,13 @@ export default {
             };
         }
 
-        // build tile
+        // build tile mesh
         geometry._count++;
         const material = new LayeredMaterial(layer.materialOptions);
         const tile = new TileMesh(geometry, material, layer, extent, level);
-        // TODO semble ne pas etre necessaire
-        tile.layers.set(layer.threejsLayer);
+
+        // TODO unecessary because is undefined...????
+        // tile.layers.set(layer.threejsLayer);
```

Comment:
`threejsLayer` is used in `Picking` and for debug isn't it ? Do you have clue on how to maybe get rid of this property ?

Reply:
It's used with visibility. This part needs to be fixed, because there's conflict with `object3d` added in `view.scene`;